kimchi/src/curve.rs

```diff
@@ -9,10 +9,14 @@ use oracle::poseidon::ArithmeticSpongeParams;
 
 ///Represents additional information that a curve needs in order to be used with Kimchi
 pub trait KimchiCurve: CommitmentCurve {
-    type OtherCurve: KimchiCurve;
+    type OtherCurve: KimchiCurve<ScalarField = Self::BaseField>;
```

Comment:
and `BaseField = Self::ScalarField` :o ?

Reply:
It wasn't necessary so I was not sure about adding it, but it can be done.
I also added a little doc.